src/components/application_manager/src/commands/command_request_impl.cc

```diff
@@ -765,30 +801,76 @@ void CommandRequestImpl::AddDissalowedParameterToInfoString(
 
 void CommandRequestImpl::AddDisallowedParametersToInfo(
     smart_objects::SmartObject& response) const {
-  std::string info;
+  LOG4CXX_AUTO_TRACE(logger_);
+  const mobile_apis::FunctionID::eType id =
+      static_cast<mobile_apis::FunctionID::eType>(function_id());
+
+  if (!helpers::Compare<mobile_apis::FunctionID::eType,
+                        helpers::EQ,
+                        helpers::ONE>(
+          id,
+          mobile_apis::FunctionID::SubscribeVehicleDataID,
+          mobile_apis::FunctionID::UnsubscribeVehicleDataID,
+          mobile_apis::FunctionID::GetVehicleDataID,
+          mobile_apis::FunctionID::SendLocationID)) {
+    LOG4CXX_INFO(logger_, "The function id: " << id << " is not supported.");
+    return;
+  }
 
-  RPCParams::const_iterator it =
-      removed_parameters_permissions_.disallowed_params.begin();
-  for (; it != removed_parameters_permissions_.disallowed_params.end(); ++it) {
-    AddDissalowedParameterToInfoString(info, (*it));
+  std::string disallowed_by_user_info;
+  InfoAdder user_info_adder(disallowed_by_user_info);
+
+  std::for_each(removed_parameters_permissions_.disallowed_params.begin(),
+                removed_parameters_permissions_.disallowed_params.end(),
+                user_info_adder);
+
+  const size_t min_number_of_disallowed_params = 1;
```

Comment:
@ZhdanovP I don't think we need this constant with a so long name here at all

Reply:
@AKalinich-Luxoft I think the long name is not problem but more descriptive. Don't think so?

Reply:
@ZhdanovP in this case do it in more obvious way. For example create
```
const bool is_single_param = 1 == removed_parameters_permissions_.count();
```
and move it under `if (!disallowed_by_user_info.empty()) ` condition